README.md

```diff
@@ -61,82 +69,74 @@ that can be used as a dependency in any other wallet based project. It is develo
+Once your changes have been pushed to your forked repository, you can create a pull request to propose integrating your updates into the main project.  
+For guidance on creating a pull request, please visit the [Making a Pull Request](https://github.com/openMF/mobile-wallet/wiki/Making-a-Pull-Request) guide.
 
 ### **Squashing Your Commits**
+To ensure a clean and organized Git history, contributors are encouraged to squash their commits before merging.  
+Instructions on how to squash commits can be found in the [Squashing Your Commits](https://github.com/openMF/mobile-wallet/wiki/Squashing-Your-Commits) guide.
 
-To maintain a clean and organized Git history, it is recommended to squash your commits before merging.  
-Don't know how to squash commits? Click [here](https://github.com/openMF/mobile-wallet/wiki/Squashing-Your-Commits) to learn more.
-
-### **Solving Merge Conflicts**
-
-Sometimes, your pull request may encounter merge conflicts that need to be resolved manually.  
-Don't know how to resolve merge conflicts? Click [here](https://github.com/openMF/mobile-wallet/wiki/Solving-Merge-Conflicts) to learn more.
+### **Resolving Merge Conflicts**
+Occasionally, merge conflicts may arise when your pull request is being reviewed. These conflicts need to be resolved manually.  
+To learn how to resolve merge conflicts, please refer to the [Solving Merge Conflicts](https://github.com/openMF/mobile-wallet/wiki/Solving-Merge-Conflicts) guide.
 
 ### Conclusion
 By following these contribution guidelines, you're all set to start contributing to the Mobile Wallet (Mifos Pay) project. We appreciate your efforts and look forward to your valuable contributions. Happy coding!
 
-## Instructions to get the latest APK
+## Instructions to Get the Latest APK
 
-To get the latest apk fom the Github actions artifacts, follow these steps:
+To get the latest APK from the GitHub Actions artifacts, follow these steps:
 
-1. Navigate to the [Actions](https://github.com/openMF/mobile-wallet/actions?query=workflow%3A%22Mobile-Wallet+CI%5BMaster%2FDev%5D%22+event%3Apush) tab of this repository.
-2. Click the latest workflow from the workflows list.
-3. Scroll down to the **Artifacts** section and click the **mobile-wallet** hyperlink.
-4. After successful download, extract the zip file to your preferred location.
+1. Navigate to the [Actions](https://github.com/openMF/mobile-wallet/actions?query=workflow%3A%22Mobile-Wallet+CI%5BMaster%2FDev%5D%22+event%3Apush) tab of this repository.  
 
-## Wiki
+2. Click the latest workflow from the workflows list.  
+
+3. Scroll down to the **Artifacts** section, where you will find: **Android APKs** | **Linux-App** | **MacOS-App** | **Windows-Apps**. Download all the available zip files.  
+  <img height='200' src="https://github.com/user-attachments/assets/ab7783ff-3834-4be2-9ce8-6342746b22a2">
 
+  
+
+5. After successful download, extract the zip files to your preferred location.  
+
```

Comment:
We do not need an installation guide, people know how to install the application, remove those guides

Reply:
Removed Installation Guides. Is everything ok now